pkg/storage/engine/mvcc_test.go

```diff
@@ -3222,6 +3287,11 @@ func TestFindValidSplitKeys(t *testing.T) {
 			ms := &enginepb.MVCCStats{}
 			val := roachpb.MakeValueFromString(strings.Repeat("X", 10))
 			for _, k := range test.keys {
+				if i == 7 {
```

Comment:
why is 7 special cased? deserves a comment if this is intentional

Reply:
Heh, that definitely wasn't supposed to be left in. Thanks!